src/PIL/ImageTk.py

```diff
@@ -184,6 +184,7 @@ def paste(self, im, box=None):
         :param im: A PIL image. The size must match the target region.  If the
                    mode does not match, the image is converted to the mode of
                    the bitmap image.
+        :param box: Deprecated.
```

Comment:
```suggestion
        :param box: Deprecated. This parameter will be removed in Pillow 10
                    (2023-07-01).
```
Shall we include the removal date in the docs?

Reply:
Yes good idea!